src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -67,4 +67,9 @@ public CommandResult execute() {
         }
     }
 
+    @Override
+    public boolean isMutating(){
+        return MUTATING;
+    }
+
```

Comment:
This method should have a header comment. Only then users of the overridden method will know how it exactly behaves

Similar to other isMutating() method overriden in other child classes.